README.md

```diff
@@ -17,7 +17,7 @@ project for the software development community to use in their applications.
 
 If you want to see Hypertrace in action, you can quickly start Hypertrace via Docker. 
 
-:IMPORTANT: **We recommend you change the [Docker Desktop default settings](https://hypertrace-docs.s3.amazonaws.com/docker-desktop.png) from `2 GB` of memory to `4 GB` of memory, and set CPUs to at least 4 CPUs.** When reporting problems, please include the output of `docker stats --no-stream`.
+`Note:` **We recommend you change the [Docker Desktop default settings](https://hypertrace-docs.s3.amazonaws.com/docker-desktop.png) from `2 GB` of memory to `4 GB` of memory, and set CPUs to at least 4 CPUs.** When reporting problems, please include the output of `docker stats --no-stream`.
```

Comment:
We could introduce issue templates, that will list what is necessary to attach when reporting an issue.

Reply:
:+1: let's address this in another PR.

Reply:
I guess it's addressed by #82 !